src/adapter/src/catalog/builtin.rs

```diff
@@ -2371,6 +2371,67 @@ JOIN mz_catalog.mz_roles role_owner ON role_owner.id = class_objects.owner_id
 WHERE mz_schemas.database_id IS NULL OR d.name = pg_catalog.current_database()",
 };
 
+pub const PG_DEPEND: BuiltinView = BuiltinView {
+    name: "pg_depend",
+    schema: PG_CATALOG_SCHEMA,
+    sql: "CREATE VIEW pg_catalog.pg_depend AS WITH class_objects AS (
+    SELECT
+        id,
+        CASE
+            WHEN type = 'table'  THEN 'pg_tables'::pg_catalog.regclass::pg_catalog.oid
+            WHEN type = 'source' THEN 'pg_tables'::pg_catalog.regclass::pg_catalog.oid
+            WHEN type = 'view'   THEN 'pg_views'::pg_catalog.regclass::pg_catalog.oid
+            WHEN type = 'materialized-view' THEN 'pg_matviews'::pg_catalog.regclass::pg_catalog.oid
+        END classid,
+        schema_id
+    FROM mz_catalog.mz_relations
+    UNION ALL
+    SELECT mz_indexes.id, mz_indexes.oid, schema_id
```

Comment:
Should, `mz_indexes.oid` be `'pg_indexes'::pg_catalog.regclass::pg_catalog.oid`?

Reply:
good catch